relays/client-substrate/src/error.rs

```diff
@@ -138,13 +138,11 @@ impl Error {
 impl MaybeConnectionError for Error {
 	fn is_connection_error(&self) -> bool {
 		match *self {
-			Error::RpcError(RpcError::Transport(_))
-				// right now if connection to the ws server is dropped (after it is already established),
-				// we're getting this error
-				| Error::RpcError(RpcError::Internal(_))
```

Comment:
There's no internal anymore, or has it been renamed?

Reply:
It was removed: https://github.com/paritytech/jsonrpsee/commit/1a2a19906ae00b9e19d67b14595610e7fb98091d#diff-363a23d98a6c2addecef2665cf68deec178a1dc19d7b625439d4cc50f7f4af50